wakuv2/waku.go

```diff
@@ -1600,16 +1583,17 @@ func (w *Waku) seedBootnodesForDiscV5() {
 			}
 		// If we go online, trigger immediately
 		case <-w.connectionChanged:
-			if canQuery() {
-				err := w.restartDiscV5()
-				if err != nil {
-					w.logger.Warn("failed to restart discv5", zap.Error(err))
-				}
+			if w.cfg.EnableDiscV5 {
```

Comment:
I am wondering rather than having this check here, we can avoid altogether starting the routine `seedBootnodesForDiscV5` itself in case of lightNode and start it only if discv5 is enabled. 

For lightNodes, we can have another routine that waits onConnectionChanged and take different action such as do a StoreNode Query (if it is not already done).

Reply:
This can be taken up in a separate PR as well.